src/hotspot/share/opto/superword.cpp

```diff
@@ -1742,7 +1744,27 @@ VTransformBoolTest PackSet::get_bool_test(const Node_List* bool_pack) const {
       mask = bol->_test.negate();
       is_negated = true;
     }
-  }
+    break;
+  }
+  case Op_CmpU:
+  case Op_CmpUL:
+  case Op_CmpU3:
+  case Op_CmpUL3:
```

Comment:
Do these really ever survive to here? I thought they should all be turned into other ops? Do you have an example?

Reply:
will fix.